src/component/Bpanel.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Button from './Button';
+
+const Bpanel = () => (
+  <div>
+    <Button />
+  </div>
+);
```

Comment:
Therefore you will have to call <Button /> component here as many times as your calculator buttons , [see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/buttonPanel/buttons.png)